docs/components/authentication/sign-in.mdx

```diff
@@ -92,6 +92,13 @@ All props are optional.
 
   ---
 
+  - `withSignUp?`
+  - `boolean`
+
+  Opt into sign-in-or-up flow by setting this prop to `true`. Defaults to `false`. When true, if a user does not exist, they will be prompted to sign up. If a user exists, they will be prompted to sign in.
```

Comment:
Maybe I am misunderstanding how you trigger this flow, but isn't it supposed default to sing-in -> sign-up flow now?

Reply:
It is if no signUpUrl is defined. If it is, this prop allows you to optionally opt into the flow at the component level too.